<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModemStatusView</class>
 <widget class="QWidget" name="ModemStatusView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>316</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modem Status</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="labelManufacturerCaption">
     <property name="text">
      <string>Manufacturer:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="labelManufacturer"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelModelCaption">
     <property name="text">
      <string>Model:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="labelModel"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelSerialNumberCaption">
     <property name="text">
      <string>Serial number:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="labelSerialNumber"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelFirmwareCaption">
     <property name="text">
      <string>Firmware:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="labelFirmware"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="labelCellularClassCaption">
     <property name="text">
      <string>Cellular class:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="labelCellularClass"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="labelVoiceClassCaption">
     <property name="text">
      <string>Voice class:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="labelVoiceClass"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="labelDataClassCaption">
     <property name="text">
      <string>Data class:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="labelDataClass"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="labelCustomDataClassCaption">
     <property name="text">
      <string>Custom data class:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLabel" name="labelCustomDataClass"/>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="labelReadyStateCaption">
     <property name="text">
      <string>Ready state:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLabel" name="labelReadyState"/>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="labelEmergencyModeCaption">
     <property name="text">
      <string>Emergency mode:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QLabel" name="labelEmergencyMode"/>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="labelRegisterStateCaption">
     <property name="text">
      <string>Register state:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QLabel" name="labelRegisterState"/>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="labelProviderNameCaption">
     <property name="text">
      <string>Provider name:</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QLabel" name="labelProviderName"/>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="labelSignalStrengthCaption">
     <property name="text">
      <string>Signal strength:</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QLabel" name="labelSignalStrength"/>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="labelConnectionIdCaption">
     <property name="text">
      <string>Connection ID:</string>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <widget class="QLabel" name="labelConnectionId"/>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="labelInterfaceIdCaption">
     <property name="text">
      <string>Interface ID:</string>
     </property>
    </widget>
   </item>
   <item row="14" column="1">
    <widget class="QLabel" name="labelInterfaceId"/>
   </item>
   <item row="15" column="0">
    <widget class="QLabel" name="labelConnectionStateCaption">
     <property name="text">
      <string>Connection state:</string>
     </property>
    </widget>
   </item>
   <item row="15" column="1">
    <widget class="QLabel" name="labelConnectionState"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
